src/lib/tracer.js

```diff
@@ -1,8 +1,16 @@
 import Tracer from "datadog-tracer"
+import config from "config"
 import { forIn, has } from "lodash"
 import { getNamedType, GraphQLObjectType } from "graphql"
 import * as introspectionQuery from "graphql/type/introspection"
 
+const { DD_TRACER_SERVICE_NAME, DD_TRACER_HOSTNAME } = config
```

Comment:
We should rename `metaphysics-k8s` to `metaphysics` in this ENV var on the new stacks (although I'm fine leaving it as-is, with the `-k8s`).

Reply:
Will rename once this is merged